openedx/core/djangoapps/appsembler/analytics/context_processors.py

```diff
@@ -4,13 +4,18 @@
 from student.auth import user_has_role
 from student.roles import CourseCreatorRole
 
-from openedx.core.djangoapps.appsembler.analytics.helpers import should_show_hubspot
+from openedx.core.djangoapps.appsembler.analytics.helpers import (
+    should_show_hubspot
+)
 
 
 def google_analytics(request):
+    app_id = getattr(settings, 'GOOGLE_ANALYTICS_APP_ID', None)
+    show_app = True if app_id else False
+
     data = {
-        'GOOGLE_ANALYTICS_APP_ID': getattr(settings, 'GOOGLE_ANALYTICS_APP_ID', None),
-        'SHOW_GOOGLE_ANALYTICS': getattr(settings, 'GOOGLE_ANALYTICS_APP_ID', None),
+        'GOOGLE_ANALYTICS_APP_ID': app_id,
+        'SHOW_GOOGLE_ANALYTICS': show_app,
```

Comment:
`show_app = True if app_id else False` doesn't seem to be Pythonic. `bool()` should do the same job.

```suggestion
        'SHOW_GOOGLE_ANALYTICS': bool(show_app),
```